tests/resources/functions/package-dart.sh

```diff
@@ -0,0 +1,14 @@
+
+echo 'Dart Packaging...'
+
+cp -r $(pwd)/tests/resources/functions/dart $(pwd)/tests/resources/functions/packages/dart
+
+docker run --rm -v $(pwd)/tests/resources/functions/packages/dart:/app -w /app appwrite/env-dart-2.10:1.0.0 ls
+docker run --rm -v $(pwd)/tests/resources/functions/packages/dart:/app -w /app --env PUB_CACHE=./.appwrite appwrite/env-dart-2.10:1.0.0 pub get
+docker run --rm -v $(pwd)/tests/resources/functions/packages/dart:/app -w /app --env PUB_CACHE=./.appwrite appwrite/env-dart-2.10:1.0.0 pub get --offline
```

Comment:
This command isn't working for me. Used the non --offline version

Reply:
Yeah, was leftover of syncing this fork to my linux notebook.